src/views/News/components/NewsItem/index.js

```diff
@@ -6,44 +6,115 @@ import Collapse from '@material-ui/core/Collapse';
 import Grid from '@material-ui/core/Grid';
 import Button from '@material-ui/core/Button';
 import newsService from './../../../../services/news';
+import EditIcon from '@material-ui/icons/Edit';
+import storage from '../../../../services/storage';
 import DeleteIcon from '@material-ui/icons/Delete';
 import { Link } from 'react-router-dom';
 
 import './newsItem.scss';
 
-//Switched to table rows
+
+
 export default class NewsItem extends Component {
   constructor(props) {
     super(props);
 
     this.state = {
+      // for collapsable news postings
       open: false,
       posting: props.posting,
       snackbarOpen: false,
       snackbarMessage: 'Something went wrong',
     };
     
```

Comment:
Love the use of the Snackbar. It really helps the user know what's going on when things fail haha